aten/src/ATen/native/cpu/UpSampleKernelAVXAntialias.h

```diff
@@ -283,7 +306,7 @@ void upsample_avx_bilinear_uint8(
         F::compute_indices_int16_weights_aa(
             /*input_size=*/yin,
             /*output_size=*/yout,
-            /*stride=*/1,
+            /*stride=*/xout,
```

Comment:
Can you add a test that catches this, and should you submit to the release branch?

Reply:
Oh I see, I thought this was a bugfix.